### PR TITLE
Remote playbar state

### PR DESCRIPTION
**Pull request by reidbaker**
This corrects the playbar state on the host when guests connect.

### CODE REVIEW

File: SoundStream/src/com/lastcrusade/soundstream/service/PlaylistService.java [hunk 1]
```diff
@@ -73,20 +74,23 @@ public PlaylistService getService() {
 
     private BroadcastRegistrar    registrar;
     private IPlayer               mThePlayer;
-    private SingleFileAudioPlayer mAudioPlayer;
+    private SingleFileAudioPlayer mAudioPlayer; //TODO remove this when we add stop to IPlayer
```
Comment (theJenix) [inline SoundStream/src/com/lastcrusade/soundstream/service/PlaylistService.java:77]: For the record, we need to add stop and "setSong", which I'm ok doing...as long as we comment it as a no-op in RemoteAudioPlayer.

File: SoundStream/src/com/lastcrusade/soundstream/service/PlaylistService.java [hunk 2]
```diff
@@ -73,20 +74,23 @@ public PlaylistService getService() {
...
+    private boolean isLocalPlayer;
 
     @Override
     public IBinder onBind(Intent intent) {
         //create the local player in a separate variable, and use that
         // as the player until we see a host connected
         this.mAudioPlayer  = new SingleFileAudioPlayer(this, (CustomApp)this.getApplication());
-        this.mThePlayer    = mAudioPlayer;
+        //Assume we are local until we connect to a host
+        isLocalPlayer = true;
```
Comment (theJenix) [inline SoundStream/src/com/lastcrusade/soundstream/service/PlaylistService.java]: Can you line up the = signs?

**Comment by theJenix** [comment]
Other than the misaligned = sign, this looks like :money_with_wings:

**Comment by reidbaker** [comment]
Fixed spacing. I have a thought about how we should change IPlayer lets talk when you get a chance

**Comment by dgreenhalgh** [comment]
:shipit:

**Pull request merged**